datafusion/benches/parquet_query_sql.rs

```diff
@@ -229,7 +229,7 @@ fn criterion_benchmark(c: &mut Criterion) {
         });
     }
 
-    // Clean up temporary file if any
+    // Temporary file must outlive the benchmarks, it is deleted when dropped
```

Comment:
This was review feedback from https://github.com/apache/arrow-datafusion/pull/1738#discussion_r807047918

Reply:
sorry I needed the dopamine hit of clicking the green button -- I should have waited